com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/ClassLoaderResourceScanner.java

```diff
@@ -66,7 +67,7 @@ private static void scanDir(File dir, String filter, Set<String> results) {
 
     private static void scanJar(URL resource, String filter, Set<String> results) throws IOException {
         String resPath = resource.getPath();
-        String jarPath = resPath.replaceFirst("[.]jar[!].*", ".jar").replaceFirst("file:", "");
+        String jarPath = URLDecoder.decode(resPath.replaceFirst("[.]jar[!].*", ".jar"), "UTF-8").replaceFirst("file:", "");
```

Comment:
@dri-richard could you please add a comment here about why we URL decode the path?